app/main.py

```diff
@@ -1,3 +1,31 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
```

Comment:
Use double quotes for consistency in your code as per the checklist requirement.